plugins/modules/zos_job_query.py

```diff
@@ -129,17 +139,6 @@
             Return code or abend resulting from the job submission.
           type: str
           sample: CC 0000
-        msg_code:
```

Comment:
I'm curious why these attributes were removed from the module doc, were they never returned to begin with?

Reply:
That is the reason, as of now, we do not return these values. So, for version 1.9.x I am removing them since we won't have any enhancements for this version. However, we can let them in the docs and if somebody needs them then add them as a bug fix.

Now that I'm reading this I'm tempted to keep them just in case.

@ketankelkar  @ddimatos  FYI

Reply:
Added this back.